src/scripts/modules/media/body/body.less

```diff
@@ -64,8 +64,12 @@ body.cc-opened {
   .concept-coach-wrapper {
     text-align: left;
   }
+
   [data-math]:not(.math-rendered) {
-    display: none;
+    visibility: hidden;
+    .MathJax {
```

Comment:
"why hide the parent but not the child?"

And could you include a comment in the CSS? I don't think it would be clear looking back at the code in a couple months